deploy/kubernetes/overlays/examples/cadvisor-args.yaml

```diff
@@ -15,5 +18,5 @@ spec:
           - --max_housekeeping_interval=15s
           - --event_storage_event_limit=default=0
           - --event_storage_age_limit=default=0
-          - --disable_metrics=percpu,disk,network,tcp,udp # enable only diskIO, cpu, memory
+          - --disable_metrics=percpu,tcp,udp              # enable only diskIO, cpu, memory, network, disk
```

Comment:
Does the underlying cgroup driver support exposing these metrics ?

Reply:
The cgroup driver (cgroupfs vs systemd) shouldn't matter.  cAdvisor doesn't really care what created the cgroups, just that they exist.

To answer the question: Yes, these metrics work for both cgroupfs and systemd cgroup drivers.